src/app/shared/auth-nav-menu/user-menu/themed-user-menu.component.ts

```diff
@@ -0,0 +1,33 @@
+import {Component, Input} from '@angular/core';
+import {ThemedComponent} from '../../theme-support/themed.component';
+import {UserMenuComponent} from './user-menu.component';
+
+/**
+ * This component represents the user nav menu.
+ */
+@Component({
+  selector: 'ds-themed-user-menu',
+  templateUrl: './../../theme-support/themed.component.html',
+  styleUrls: []
+})
+export class ThemedUserMenuComponent extends ThemedComponent<UserMenuComponent>{
+
+  /**
+   * The input flag to show user details in navbar expandable menu
+   */
+  @Input() inExpandableNavbar = false;
```

Comment:
@Leano1998 thanks for this PR, just some minor feedback since #2075 `@Input()`s should not have default values, because this way we can override the default values in the themed components in the `themes` folder (more info in #2164). So could you please replace this line with `@Input() inExpandableNavbar: boolean;`

Reply:
@alexandrevryghem Thank you for your review!
I took you suggestion and removed the default value.